packages/@lwc/ssr-runtime/src/index.ts

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+const MULTI_SPACE = /\s+/g;
+
+type Attributes = Record<string, string>;
```

Comment:
```suggestion
type Attributes = Record<string, unknown>;
```
Could be anything really.